lib/kafka/broker.rb

```diff
@@ -71,7 +71,31 @@ def list_offsets(**options)
     def produce(**options)
       request = Protocol::ProduceRequest.new(**options)
 
-      send_request(request)
+      resp = send_request(request)
+
+      return if resp.nil?
+
+      errored_partitions = []
```

Comment:
Would it be possible to gate this logging code behind an `ENV` var that we can set in console? So that we can run it in prod?